polymetis/docs/source/installation.md

```diff
@@ -59,7 +56,7 @@
         make -j
         ```
 
-5. Start developing! Remember to rebuild if modifying C++ source code. When updating, you may need to do a `mamba env update --file ./polymetis/environment.yml --prune` to update dependencies.
+5. Start developing! Remember to rebuild if modifying C++ source code. When updating, you may need to do a `conda env update --file ./polymetis/environment.yml --prune` to update dependencies.
```

Comment:
Just curious, what's the problem with Mamba?

Reply:
Haha I just wanted to be consistent -- conda everywhere, and then the text about replacing conda with mamba. Just ocd